core/math/random_number_generator.h

```diff
@@ -52,6 +52,7 @@ class RandomNumberGenerator : public RefCounted {
 	_FORCE_INLINE_ void randomize() { randbase.randomize(); }
 
 	_FORCE_INLINE_ uint32_t randi() { return randbase.rand(); }
+	_FORCE_INLINE_ bool randb(real_t chance = 0.5) { return randbase.randb(chance); }
```

Comment:
```suggestion
	_FORCE_INLINE_ bool randb(real_t p_chance = 0.5) { return randbase.randb(p_chance); }
```
Prefix arguments with `p_`

Reply:
Are these changes I need to make on my side and recommit or did you make them already using GitHub magic?

Reply:
You need to make them 🙂, they are suggestions that can be applied but they're not complete and you need to fix the uses of the arguments and such